.github/workflows/ci-backend.yml

```diff
@@ -133,6 +133,11 @@ jobs:
               run: |
                   npm run schema:build:python && git diff --exit-code
 
+            - name: Check if antlr definitions are up to date
+              run: |
+                  sudo apt-get install antlr
+                  npm run antlr:generate && git diff --exit-code
```

Comment:
Maybe only to save me reading the code 🙈 what is guaranteeing these schema files are in the generated docker images?

Reply:
They're committed to git :D